modules/quality_control/test/quality_controlTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * AcknowledgementsIntegrationTest automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Wang Shen <wangshen.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+ require_once __DIR__
+    . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+/**
+ * AcknowledgementsIntegrationTest
```

Comment:
This doesn't match the class name (QualityControl vs Acknowledgements), same with the first line of this file.